### ISSUE TITLE
podman 4.2 has renamed "play kube" to "kube play" with an alias for "play kube"

### ISSUE THREAD

**Issue by richm** [containers/ansible-podman-collections#456]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

In podman 4.2, the command `play kube` has been renamed to `kube play` with an alias for `play kube`.  The `containers.podman.podman_play` module should detect which version of podman is being used, and use either `play kube` or `kube play` depending on the version e.g. at https://github.com/containers/ansible-podman-collections/blob/master/plugins/modules/podman_play.py#L143
Note that podman will continue to provide the alias `play kube` for some time, so this probably isn't urgent.

**Steps to reproduce the issue:**

1.

2.

3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version of the `containers.podman` collection:**
**Either git commit if installed from git: `git show --summary`**
**Or version from `ansible-galaxy` if installed from galaxy: `ansible-galaxy collection list | grep containers.podman`**

```
(paste your output here)
```

**Output of `ansible --version`:**

```
(paste your output here)
```

**Output of `podman version`:**

```
(paste your output here)
```

**Output of `podman info --debug`:**

``` yaml
(paste your output here)
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
(paste your output here)
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

``` yaml
(paste your output here)
```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by fsdrw08** [comment]
> Yeah, good point. Also they added new things there, so probably it might affect the whole module.

I would suggest create a new module call `podman_kube_play` independent to the old one.

**Comment by sshnaidm** [comment]
@fsdrw08 probably it's a good idea, a lot of changes came in.

**Comment by nishipy** [comment]
According to the [release note of Podman v4.3.0](https://github.com/containers/podman/releases/tag/v4.3.0), `podman generate kube` was also renamed to `podman kube generate`. Thanks to aliases, the old commands still work apparently.
> The podman generate kube and podman play kube commands have been renamed to podman kube generate and podman kube play to group Kubernetes-related commands. Aliases have been added to ensure the old command names still function.

**Comment by sshnaidm** [comment]
It was fixed in #727
